libs/ui/src/search_select.tsx

```diff
@@ -158,10 +160,10 @@ export function SearchSelect<T = string>({
       unstyled
       components={{ DropdownIndicator, MultiValueRemove }}
       className="search-select"
-      maxMenuHeight="50vh"
```

Comment:
Good catch figuring out that this wasn't working. I'm a bit concerned that making it an optional prop means that all of the other places that SearchSelect is used will be vulnerable to overflow (though apparently they already are). Maybe we set a safe default?